injective-common/substreams.yaml

```diff
@@ -1,124 +1,43 @@
-# requires substreams v1.7.1 and above
 specVersion: v0.1.0
 package:
-  name: injective_common
-  version: v0.2.3
+  name: mantra_common
+  version: v0.0.1
```

Comment:
Mmm not sure about the versioning here, let's keep the same as before to make sure we don't break stuff

Reply:
I'm pushing `injective-common` to 0.2.4 (which will be the new version) the latest version is 0.2.3